### DISCUSSION TITLE
a question to who owns this thingy

### DISCUSSION THREAD

**Discussion by GhostCyclo6286** [DEMOLITIONDON96/Demolition-Engine#74] [General (Friday Night Chattin')]
how do i update repo's?

**Comment by ThatOneSillyGuyDon** [comment]
You can’t update repos unless you’re the owner or a contributor I invited, also, I’m the owner, duh lol

**Reply by ThatOneSillyGuyDon** [reply]
In other words, you can only use PRs, for that’s where you could make changes if we accept them
